src/phoenix/core/traces.py

```diff
@@ -1,29 +1,81 @@
+import math
 from collections import defaultdict
-from typing import Iterable, Tuple, cast
+from typing import Any, Iterable, Tuple, cast
 
-from pandas import DataFrame
+import numpy as np
+import pandas as pd
 
-from phoenix.trace.schemas import SpanID
+import phoenix.trace.semantic_conventions as sc
+from phoenix.trace.schemas import ATTRIBUTE_PREFIX, SpanID
+
+NAME = "name"
+STATUS_CODE = "status_code"
+SPAN_KIND = "span_kind"
+TRACE_ID = "context.trace_id"
+SPAN_ID = "context.span_id"
+PARENT_ID = "parent_id"
+START_TIME = "start_time"
+END_TIME = "end_time"
+LATENCY_MS = "__computed__latency_ms"
+"The latency (or duration) of the span in milliseconds"
+LLM_TOKEN_COUNT_TOTAL = ATTRIBUTE_PREFIX + sc.LLM_TOKEN_COUNT_TOTAL
+LLM_TOKEN_COUNT_PROMPT = ATTRIBUTE_PREFIX + sc.LLM_TOKEN_COUNT_PROMPT
+LLM_TOKEN_COUNT_COMPLETION = ATTRIBUTE_PREFIX + sc.LLM_TOKEN_COUNT_COMPLETION
+CUMULATIVE_LLM_TOKEN_COUNT_TOTAL = "__computed__cumulative_token_count_total"
```

Comment:
Oh nice. Maybe we should put `__computed__` as a constant just to keep it clean like attribute prefix

Reply:
ideally these computed columns should be random strings to eliminate any risk of conflict with preexisting columns, so so i don't think we need to formalize the `__computed__` prefix as a variable because it exists purely as a courtesy for the human reader